react-forms/src/DatePicker.tsx

```diff
@@ -182,9 +184,19 @@ export default function DateTimePicker<T>(props: IProps<T>) {
 
             <input
                 data-help={guid}
-                className={"gpa-gemstone-datetime form-control" + (props.Valid(props.Field) ? '' : ' is-invalid')}
+                className={`gpa-gemstone-datetime form-control ${!props.Valid(props.Field) || feedbackMessage ? 'is-invalid' : ''}`}
                 type={props.Type === undefined ? 'date' : props.Type}
                 onChange={(evt) => {
+                    const inputVal = evt.target.value ?? "";
+                    const date = moment(inputVal, boxFormat, true);
+                    const validStartDate = moment("1753-01-01", "YYYY-MM-DD");
+
+                    if (date.isValid() && date.isSameOrAfter(validStartDate)) {
+                        setFeedbackMessage(""); 
+                    } else {
+                        // Date is invalid due to being before 1753
+                        setFeedbackMessage("Date cannot be before 01/01/1753");
```

Comment:
What happens if a user enters an invalid date (e.g. `13/13/2024`) That will fail the `date.isValid()` part and display this message. That's not ideal since `2024` is obviously after `1753` so I would suggest you check these two condition sepperately and set the error message appropriately (`a valid date needs to be entered` or `Date cannot be before 01/01/1753`